### PR TITLE
First version of Push Notifications

### LINKED ISSUE #28: Implement new feature to push notification for event reminder
Implement new feature to push notification for event reminder after creating an event. This feature is context aware.

### PR DESCRIPTION
**Pull request by SnowArtz**
Push Notifications base version: For now we have a new page where we can choose a Date, then press a Button and a Local Notification is schedueled. This then will be evolve to be adapted to the event creation.

https://github.com/ISIS3510-202410-Team-13/Flutter/assets/69651671/8c10f771-9764-498e-be9b-e17b9c890898

### CODE REVIEW

File: unischedule/android/app/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     namespace "com.example.unischedule"
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 34
```
Comment (SnowArtz) [inline unischedule/android/app/build.gradle:27]: For the use of local notifications I had to change this. What you think guys? @iGotty @jsurrea

File: unischedule/pubspec.yaml [hunk 2]
```diff
@@ -30,11 +30,13 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_svg: ^0.22.0
+  flutter_svg: ^2.0.10
```
Comment (SnowArtz) [inline unischedule/pubspec.yaml:33]: I updated the versions of flutter_svg, we were using a very old one.

File: unischedule/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.USE_BIOMETRIC"/>
+    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM"/>
+    <uses-permission android:name="android.permission.USE_EXACT_ALARM"/>
```
Comment (SnowArtz) [inline unischedule/android/app/src/main/AndroidManifest.xml:4]: IDK if we have to ask for managing this permissions in the users Device when deploying the app @iGotty @jsurrea

Comment (jsurrea) [reply unischedule/android/app/src/main/AndroidManifest.xml:4]: Better safe than sorry!

File: unischedule/android/app/build.gradle (hunk 1 above)
Comment (jsurrea) [reply unischedule/android/app/build.gradle:27]: LGTM 👍

**Pull request merged**